src/harshadBJun20/assignment25/SumOfNumbersInString.java

```diff
@@ -0,0 +1,28 @@
+package harshadBJun20.assignment25;
+
+/*Assignment 25 : 8th July 2020
+
+WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Ouput: Sum of all numbers in String is: 53*/
+
+public class SumOfNumbersInString {
+	
+	void calSumOfNumbers(String str) {
+		int sumOfDigit=0;
+		int digit=0;
+		for(int index=0;index<str.length();index++) {
+			if(!Character.isLetter(str.charAt(index))) {
+				String charValue = String.valueOf(str.charAt(index));
+				digit = Integer.parseInt(charValue);
+				sumOfDigit +=digit;
```

Comment:
Hi Harshad,
We don't need to initialize variables -> digit and charValue 
You can directly check the condition:
	if(Character.isDigit(str.charAt(index)))
and then add the number in sumOfDigit 
	sumOfDigit += Character.getNumericValue(str.charAt(index)));

Reply:
@HarshadBJun20 work on suggestion please

Reply:
Hi @AasthaJJun20,
Yes you are right. Thanks for the suggestions. I have made the necessary changes. Please review.